src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
     return mediaDialog;
   }
 
+  public VisualEditorPageObject  getTextArea() {
+    infoboxNode.click();
+    return this;
+  }
+
+  public VisualEditorPageObject  getInfoboxNode() {
```

Comment:
Same as above method - it needs to be either clickInfobox() or clickInfoboxNode() or be split into getInfoboxNode() with a separate click